### ISSUE TITLE
Author pages

### ISSUE THREAD

**Issue by beriberikix** [CocoaPods/cocoapods.org#149]
tl;dr request: a single page for a verified owner that lists all their Pods and some stats.

An Author page would be super-handy for folks like us who have a lot of Pods. For one, it would allow us to link to a single page. It would also allow us to highlight what is official, and make it clear what is unofficial (i.e. anything not listed on our official Author Page.)

Users should be able to get to it mainly by clicking the author's name on the inline or dedicated Pod listing.

![screen shot 2015-05-14 at 4 43 43 pm](https://cloud.githubusercontent.com/assets/178376/7644352/a227417a-fa58-11e4-9dab-0c35eeae4548.png)

There should also be a dedicated page at https://cocoapods.org/authors/{{AuthorName}}. On that page would be all Pods uploaded by that Author that is verified by Trunk. That's a critical piece. There are Pods that are uploaded by other developers that imply that they are from Google or flat-out say the author is Google Inc. when it is not. If we can say, "anything linked from _this_ author page is official and anything else is not," we can help guide people to the correct and maintained Pod.

What else would be on the Author page? A logo would be nice. As far as stats go: most recently updated pod, most popular & pan-Pod quality report would be useful (ex. % doc coverage.)

That set of features and nothing more would be huge, and allow us to guide folks to the right Pods.

**Comment by orta** [comment]
Agree totally

**Comment by orta** [comment]
So, a rough version of this exists but isn't being linked publicly yet, some tricky db joinery to get the more interesting data out. 

http://cocoapods.org/owners/4 - me
http://cocoapods.org/owners/6563 - Google SDK

**Comment by beriberikix** [comment]
Nice! Where does the image come from? The cocoadocs yaml?

**Comment by orta** [comment]
Nah, they're gravatars based on the email for the account - so most should already be set up.

**Comment by orta** [comment]
I'm calling this done, pretty URLs are a separate issue, and the latest version of the site accommodates framework only projects, so it doesn't show the tests. Pods will require a re-parse on cocoadocs, so the Google one will update with time.
